crt_portal/cts_forms/forms.py

```diff
@@ -668,6 +687,219 @@ class Meta:
+            choices=PUBLIC_OR_PRIVATE_EMPLOYER_CHOICES,
+            empty_value=None,
+            widget=UsaRadioSelect,
+            required=False,
+        )
+        self.fields['employer_size'] = TypedChoiceField(
+            choices=EMPLOYER_SIZE_CHOICES,
+            empty_value=None,
+            widget=UsaRadioSelect,
+            required=False,
+        )
+        self.fields['public_or_private_school'] = TypedChoiceField(
+            choices=PUBLIC_OR_PRIVATE_SCHOOL_CHOICES,
+            empty_value=None,
+            widget=UsaRadioSelect,
+            required=False,
+        )
+        self.fields['election_details'] = TypedChoiceField(
+            choices=ELECTION_CHOICES,
+            empty_value=None,
+            widget=UsaRadioSelect,
+            required=False,
+        )
+        self.fields['inside_correctional_facility'] = TypedChoiceField(
+            choices=CORRECTIONAL_FACILITY_LOCATION_CHOICES,
+            empty_value=None,
+            widget=UsaRadioSelect,
+            required=False,
+        )
+        self.fields['correctional_facility_type'] = TypedChoiceField(
+            choices=CORRECTIONAL_FACILITY_LOCATION_TYPE_CHOICES,
+            widget=UsaRadioSelect,
+            required=False,
+            label=POLICE_QUESTIONS['correctional_facility_type']
+        )
+        self.fields['commercial_or_public_place'] = TypedChoiceField(
+            choices=COMMERCIAL_OR_PUBLIC_PLACE_CHOICES,
+            empty_value=None,
+            widget=UsaRadioSelect,
+            required=False,
+        )
+        # incident location
+        self.fields['protected_class'] = ModelMultipleChoiceField(
+            error_messages={'required': PROTECTED_CLASS_ERROR},
+            required=False,
+            label=PROTECTED_CLASS_QUESTION,
+            queryset=ProtectedClass.objects.filter(protected_class__in=PROTECTED_CLASS_CHOICES).order_by('form_order'),
+            widget=UsaCheckboxSelectMultiple(attrs={
+                'aria-describedby': 'protected-class-help-text'
+            }),
+        )
+        self.fields['last_incident_day'].label = DATE_QUESTIONS['last_incident_day']
+        self.fields['last_incident_month'].label = DATE_QUESTIONS['last_incident_month']
+        self.fields['last_incident_year'].label = DATE_QUESTIONS['last_incident_year']
+
+        self.fields['crt_reciept_day'].label = DATE_QUESTIONS['last_incident_day']
+        self.fields['crt_reciept_month'].label = DATE_QUESTIONS['last_incident_month']
+        self.fields['crt_reciept_year'].label = DATE_QUESTIONS['last_incident_year']
+
+        self.fields['violation_summary'].widget.attrs['class'] = 'usa-textarea word-count-500'
+        self.label_suffix = ''
+        self.fields['violation_summary'].label = SUMMARY_QUESTION
+        self.fields['violation_summary'].widget.attrs['aria-describedby'] = 'details-help-text'
+        self.fields['violation_summary'].help_text = 'What did the person believe happened?'
```

Comment:
This works but I _think_ there's a good spot here to tackle the duplication of widget/field definitions across `forms.py`. A bigger effort that would span across most of the other Forms defined here -- let's tackle it in a separate issue.

Reply:
Sounds good, I was trying to do some of that but ran into some issues because I couldn't pass attributes the same way for our custom widgets and the standard widgets. I would love to have some time to clean that up a bit!